### ISSUE TITLE
NTR: Corneal transplant history (Eye structure & morphology)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7988]
New term request Corneal transplant history (samant_110204014109): Keratoplasty  UMLS: 1384683

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000096
- label: Corneal transplant history
- parents: [Past medical history (HP:0032443)](https://hpo.jax.org/app/browse/term/HP:0032443)
- pmid: 
- def: Past medical history of surgical replacement of the cornea with corneal tissue from a donor.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/7988
- synonyms: History of corneal transplantation(NONE)
